flanker/mime/message/part.py

```diff
@@ -684,7 +681,7 @@ def _base64_decode(s):
         return base64.b64decode(s)
 
     except TypeError:
-        s = s.replace("\n", "").replace("\r", "")
+        s = s.translate(_b64_translation_table).replace(LF, "")
```

Comment:
I was thinking about that but `_b64_translation_table` has to be 256 chars long and filling it up with `""` doesn't work. However it's a good point. We can create `_b64_invalid_chars` alphabet by excluding the allowed chars and use translate like that:

```python
s = s.translate(None, _b64_invalid_chars)
```